1_fetch/src/get_nlcd_LC.R

```diff
@@ -0,0 +1,97 @@
+
+download_NHD_NLCD_data <- function(sb_id,
+                                 out_path = '1_fetch/out',
+                                 downloaded_data_folder_name = NA,
+                                 create_LandCover_folder = T,
+                                 overwrite_download = T){
+  
+  #' @description download Land Cover data to the repo's fetch/src folder.
+  #' 
+  #' @param sb_id: The ScienceBase Id for the NHD-NLCD land cover dataset of interest. Str or vector of str.
+  #' @param out_path: Output folder for the downloaded data packages from ScienceBase. Default `1_fetch/out`. If path does not exist in directory, path will be created. 
+  #' @param downloaded_data_folder_name: Sub-folder name(s) for the data in out_path. Str or vector of str. Default NA, where folder names will be labelled by sb_id. If not = NA, Must be same length as sb_id. 
+  #' @param create_LandCover_folder: Create a catch all Land Cover Data Folder in 1_fetch/out where all downloaded Land Cover Data will reside. Default TRUE.
+  #' @param overwrite_download If true, data will overwrite previous download in same out_path. Default TRUE.
+  #' @value 
+  #' @example  download_NHD_NLCD_data(sb_id = '57855ddee4b0e02680bf37bf', out_path = '1_fetch/out', downloaded_data_folder_name = 'LandCover_ripbuffer_id_11')
+  #' @example  download_NHD_NLCD_data(sb_id = c('57855ddee4b0e02680bf37bf','570577fee4b0d4e2b7571d7b'), out_path = '1_fetch/out', downloaded_data_folder_name = c('LandCover_ripbuffer_id_11', 'pct_imperviousness_ripzone_id_11'))
+  
+  ## Create Land Cover Data sub folder in base directory if true
+  if(create_LandCover_folder == T){
+    out_path <- file.path(out_path, "LandCover_Data")
+    dir.create(out_path, showWarnings = F)
+  }
+  
+  ## Check lengths of sb_id and downloaded_data_folder_name
+  ## first, avoiding error when downloaded_data_folder_name = NA (checking len)
+  if(!(is.na(downloaded_data_folder_name) && length(downloaded_data_folder_name) == 1)){
+    if(length(downloaded_data_folder_name) != length(sb_id)){
+      stop('Downloaded_data_name must be same list length as sb_id')
```

Comment:
A really minor suggestion - I interpreted "list" to mean a list data structure like in R. Could just omit "list" in the returned message here (there may be other places in the script where a similar comment applies)

Reply:
Oops, I see Jared already made a comment similar to this one.